de-lib/src/main/java/org/iplantc/de/apps/client/views/list/widgets/AppTypeFilterCombo.java

```diff
@@ -0,0 +1,67 @@
+package org.iplantc.de.apps.client.views.list.widgets;
+
+import org.iplantc.de.apps.client.events.AppTypeFilterChangedEvent;
+import org.iplantc.de.client.events.EventBus;
+import org.iplantc.de.client.models.AppTypeFilter;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+import com.sencha.gxt.cell.core.client.form.ComboBoxCell;
+import com.sencha.gxt.data.shared.StringLabelProvider;
+import com.sencha.gxt.widget.core.client.form.SimpleComboBox;
+
+import java.util.Arrays;
+
+/**
+ * Created by sriram on 7/3/18.
+ */
+public class AppTypeFilterCombo implements IsWidget {
+
+    private SimpleComboBox<org.iplantc.de.client.models.AppTypeFilter> filterTypeCombo;
```

Comment:
I may have missed something - Is there a reason why this is expanded to `org.iplantc.de.client.models.AppTypeFilter` instead of just `AppTypeFilter`?

Reply:
Not Sure why it did that. But I fixed. Thanks for the review!